README.md

```diff
@@ -9,13 +9,168 @@ DUH component export to Scala
 npm i duh-scala
 ```
 
+To export a scala blackbox wrapper and attach method. This is a scala `LazyModule`
```

Comment:
This is an incomplete sentence. Also, do you think you could format the word `attach` as monospace code? When I first read the sentence my brain tried parsing it as the normal English word "attach" rather than the code concept.

Also, can we properly capitalize the word Scala everywhere in English prose?